android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/WorkerBinder.kt

```diff
@@ -37,6 +38,9 @@ import kotlinx.coroutines.launch
 /** Helper class to bind to an interactor's lifecycle to translate it to a [Worker] lifecycle. */
 public object WorkerBinder {
 
+  internal val Worker.bindingCoroutineContext: CoroutineContext
```

Comment:
I'd prefer to have it as top level in the file (outside WorkerBinder object). I suspect then it can be `private`. If it can't be private even if top-level, consider it as nit and I'll approve as is

Reply:
Oh I see, yes definitely can move it up